### PR TITLE
Feature - Release tag to config.json file

### PR DESCRIPTION
**Pull request by JoaoDanielRufino**
Signed-off-by: JoaoDanielRufino <joaodaniel0405@gmail.com>

### Description

Added template release to config.json. The CLI will replace the {tag} with the formula template repo release. This will be interesting for template versioning. If a language changes its way of building, we ask for the user to update the template version. 

  ```
{
  "dockerImageBuilder": "ritclizup/rit-go-builder",
  "templateRelease:": "{tag}",
  "inputs": [
    {
      "name": "rit_pontomais_login",
      "type": "CREDENTIAL_PONTOMAIS_LOGIN"
    },
    {
      "name": "rit_pontomais_password",
      "type": "CREDENTIAL_PONTOMAIS_PASSWORD"
    }
  ]
}
```

### How to verify it
Create a formula and check `config.json` file to see the release tag

### CODE REVIEW

**Comment by henriquemoraeszup** [comment]
Hey, can you add more description, what is this for? Is this for template versioning?

File: pkg/formula/creator/modifier/templateRelease.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
```
Comment (lucasdittrichzup) [inline pkg/formula/creator/modifier/templateRelease.go:1]: the filename is out of the project's standard (`template_release.go`)

File: pkg/formula/creator/modifier/templateRelease_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (lucasdittrichzup) [inline pkg/formula/creator/modifier/templateRelease_test.go:1]: same thing as the previous comment (`template_release_test.go`)

File: pkg/formula/creator/modifier/templateRelease_test.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+				b: []byte(`{tag}`),
+			},
+			want: []byte(tag.Name),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			tr := TemplateRelease{}
+			if got := tr.modify(tt.args.b); !reflect.DeepEqual(got, tt.want) {
```
Comment (lucasdittrichzup) [inline pkg/formula/creator/modifier/templateRelease_test.go:52]: as we adopted **testify** for testing, I suggest that you use the **asserts** of this lib

**Review by lucasdittrichzup — APPROVED** [review]

**Review by brunasilvazup — APPROVED** [review]
If this PR is part of a issue, it would be nice to attach it too.

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**